specs/commandline/verify.md

```diff
@@ -0,0 +1,99 @@
+
+An example of `trustpolicy.json`:
+
+```jsonc
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            // Policy for all artifacts, from any registry location.
+            "name": "wabbit-networks-images",  // Name of the policy.
+            "registryScopes": [ "*" ],         // The registry artifacts to which the policy applies.
+            "signatureVerification": {         // The level of verification - strict, permissive, audit, skip.
+                "level": "strict"
+            },
+            "trustStores": [ "ca:wabbit-networks" ], // The trust stores that contains the X.509 trusted roots.
+            "trustedIdentities": [                   // Identities that are trusted to sign the artifact.
+                "x509.subject: C=US, ST=WA, L=Seattle, O=wabbit-networks.io, OU=Finance, CN=SecureBuilder"
+            ]
+        }
+    ]
+}
+```
+
+In this example, only one policy is configured with the name `wabbit-networks-images`. With the value of property `registryScopes` set to `*`, this policy applies to all artifacts from any registry location. User can configure multiple trust policies for different scenarios. See [Trust Policy Schema and properties](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-policy) for details.
+
+### Verify signatures on a container image stored in a registry (Neither trust store nor trust policy is configured yet)
+
+```shell
+
+# Prerequisites: Signatures are stored in a registry referencing the signed container image
+
+# Configure trust store by adding a certificate file into trust store named "wabbit-network" of type "ca"
+notation certificate add --type ca --store wabbit-networks wabbit-networks.crt
+
+# Configure trust policy by creating a JSON document named "trustpolicy.json" under directory "{NOTATION_CONFIG}"
+# Example on Linux
+cat <<EOF > $HOME/.config/notation/trustpolicy.json
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "wabbit-networks-images",   // Name of the policy.
+            "registryScopes": [ "registry.wabbit-networks.io/software/net-monitor" ],          // The registry artifacts to which the policy applies.
+            "signatureVerification": {          // The level of verification - strict, permissive, audit, skip.
+                "level" : "strict" 
+            },
+            "trustStores": [ "ca:wabbit-networks" ], // The trust stores that contains the X.509 trusted roots.
+            "trustedIdentities": [                   // Identities that are trusted to sign the artifact.
+                "x509.subject: C=US, ST=WA, L=Seattle, O=wabbit-networks.io, OU=Finance, CN=SecureBuilder"
+            ]
+        }
+    ]
+}
+EOF
+
+# Verify signatures on a container image 
+notation verify registry.wabbit-networks.io/software/net-monitor:v1
+```
+
+### Verify signatures on an OCI artifact stored in a registry (Trust store and trust policy are configured properly)
```

Comment:
This is confusing as to OCI artifact versus trust * configured versus not.
I'd consider removing the distinction of OCI artifact per section.  Place the tag AND digest verification options for image & OCI artifact in the same properly configured section.  

I'd then have a section on what the experience is like when trust store and policy is not configured.  I'd imagine you wouldn't get a verification, but show some error message(s).

Reply:
I will create a new PR to address this comment and the error handling parts.